tests/unittests/tests-spiffs/tests-spiffs.c

```diff
@@ -382,6 +385,58 @@ static void tests_spiffs_statvfs(void)
     TEST_ASSERT(sizeof(buf) <= (stat1.f_bavail - stat2.f_bavail));
 }
 
+static void tests_spiffs_partition(void)
+{
+    vfs_umount(&_test_spiffs_mount);
+
+    spiffs_desc.base_addr = _dev->page_size * _dev->pages_per_sector;
+    spiffs_desc.block_count = 2;
+    mtd_erase(_dev, 0, _dev->page_size * _dev->pages_per_sector * _dev->sector_count);
+
+    int res = vfs_format(&_test_spiffs_mount);
+    TEST_ASSERT_EQUAL_INT(0, res);
+
+    res = vfs_mount(&_test_spiffs_mount);
+    TEST_ASSERT_EQUAL_INT(0, res);
+
+#if SPIFFS_USE_MAGIC
+    /* if SPIFFS_USE_MAGIC is used, a magic word is written in each sector */
+    uint8_t buf[4];
+    const uint8_t buf_erased[4] = {0xff, 0xff, 0xff, 0xff};
+    int read;
```

Comment:
Change the name to nread to avoid name collision with the posix read function

Reply:
done